src/UglyToad.PdfPig/Content/Page.cs

```diff
@@ -41,6 +42,11 @@ public class Page
         /// </summary>
         public IReadOnlyList<Letter> Letters => Content?.Letters ?? new Letter[0];
 
+        /// <summary>
+        /// The set of <see cref="PdfPath"/>s drawn by the PDF content.
+        /// </summary>
+        public List<PdfPath> Paths => Content?.Paths ?? new List<PdfPath>();
```

Comment:
As a general rule I avoid exposing the concrete types `List<T>` and `T[]` on the public API. This is because the consumer can mutate the content of those types and potentially break the code using those values.

Most of the time `IReadOnlyList<T>`  provides strict enough guarantees for a public API and also means we can change the internal code to use a list, array, immutablearray, etc.

This would work like this:
```suggestion
        public IReadOnlyList<PdfPath> Paths => Content?.Paths ?? new List<PdfPath>();
```

Reply:
It might be worth exposing this through the `ExperimentalAccess` member of `Page` for now. I'm still not sure what the right API for these is (for example I was planning on using lines to support underlined text detection as well as incorporating color).